### PR TITLE
Parser: Handlebars integration

### PR DESCRIPTION
**Pull request by nishantwrp**
- Implement `parser` using handlebars.
- Made sure all the current templates are compatible with this plugin.

### CODE REVIEW

File: src/parser.ts [hunk 1]
```diff
@@ -1,3 +1,29 @@
-export const parseTemplate = async (template: string): Promise<string> => {
-    return template;
```
Comment (CalebJohn) [inline src/parser.ts:2]: I'm going to miss the old parser. Simple, but it works.

**Review by CalebJohn — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -20,7 +20,9 @@
     "eslint": "^7.27.0",
     "fs-extra": "^9.0.1",
     "glob": "^7.1.6",
+    "handlebars": "^4.7.7",
     "html-entities": "^2.3.2",
+    "moment": "^2.29.1",
```
Comment (laurent22) [inline package.json:25]: By the way is there a reason why all the dependencies are under devDependencies? Doesn't it mean that the libs won't be packaged with the plugin when you publish?

Comment (nishantwrp) [reply package.json:25]: Won't webpack automatically include it in the bundled js?

Comment (laurent22) [reply package.json:25]: I don't know what webpack does, but devDependencies as the name says is for development - so for example the TypeScript package, eslint, etc. However packages you actually used in the plugin such as  handlebars or moment should be under the "dependencies" key.

Comment (nishantwrp) [reply package.json:25]: Oh ok, I'll create a PR for correcting this.

**Pull request merged**